### PR TITLE
Adding MSDE PBs for isolation and unisolation for devices

### PR DESCRIPTION
**Pull request by ssokolovich**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [#42389](https://github.com/demisto/etc/issues/42389)

## Description
Added new files: 
- MSDE playbooks for isolation and unisolation.
Modified:
- Common playbooks for isolation and unisolation.

## Screenshots
Microsoft_Defender_For_Endpoint_-_Unisolate_Endpoint_Thu_Mar_03_2022
Microsoft_Defender_For_Endpoint_-_Isolate_Endpoint_Thu_Mar_03_2022
--------------

Isolate_Endpoint_-_Generic_V2
Unisolate_Endpoint_-_Generic

## Minimum version of Cortex XSOAR
- [X] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by David-BMS — COMMENTED** [review]
- [x] Please add section headers to the MSDE playbooks

- [ ] Consider adding incident fields for what was isolated and wasn't etc.

- [x] Consider adding the isolation type as a playbook input as its support in the command

- [ ] Make sure to use the same terminology (Active Devices VS Non-valid Devices, Device active (single) vs non-active devices (plural))

For the how the process of getting the host details, let's validate when this can be developed, as its low effort maybe it can be added soon

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-Microsoft_Defender_For_Endpoint_-_Isolate_Endpoint_README.md
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-Microsoft_Defender_For_Endpoint_-_Isolate_Endpoint_README.md:32]: Missing inputs

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-Microsoft_Defender_For_Endpoint_-_Isolate_Endpoint_README.md
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-Microsoft_Defender_For_Endpoint_-_Isolate_Endpoint_README.md:39]: Type should be string for all of them

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-Microsoft_Defender_For_Endpoint_-_Unisolate_Endpoint_README.md
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-Microsoft_Defender_For_Endpoint_-_Unisolate_Endpoint_README.md:32]: Missing inputs

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-Microsoft_Defender_For_Endpoint_-_Unisolate_Endpoint_README.md
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-Microsoft_Defender_For_Endpoint_-_Unisolate_Endpoint_README.md:39]: Type string for all

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_4_0.md
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_4_0.md:4]: You have a *

Comment (David-BMS) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_4_0.md:4]: It will isolate by the device ID but it can also manage IP, Hostname

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_4_0.md
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_4_0.md:9]: *

**Review by David-BMS — CHANGES_REQUESTED** [review]
Remove not needed RN

File: Packs/MicrosoftDefenderAdvancedThreatProtection/pack_metadata.json
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/pack_metadata.json:17]: Not sure its suppose to be on MPv2
Please validate with @anara123

Comment (ssokolovich) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/pack_metadata.json:17]: Was added by the find-dependencies comamnd. Not sure why... Will remove this.

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-Microsoft_Defender_For_Endpoint_-_Isolate_Endpoint.yml
Comment (David-BMS) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-Microsoft_Defender_For_Endpoint_-_Isolate_Endpoint.yml:1052]: 6.2

File: Packs/CommonPlaybooks/Playbooks/playbook-Unisolate_Endpoint_-_Generic.yml
Comment (David-BMS) [inline Packs/CommonPlaybooks/Playbooks/playbook-Unisolate_Endpoint_-_Generic.yml:285]: Make sure that the new playbooks have skip if unavailable set to true
for isolate and unisolate

**Review by David-BMS — APPROVED** [review]

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-Microsoft_Defender_For_Endpoint_-_Isolate_Endpoint.yml
Comment (ssokolovich) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-Microsoft_Defender_For_Endpoint_-_Isolate_Endpoint.yml:1052]: Changed + also on the RN

File: Packs/CommonPlaybooks/Playbooks/playbook-Unisolate_Endpoint_-_Generic.yml
Comment (ssokolovich) [reply Packs/CommonPlaybooks/Playbooks/playbook-Unisolate_Endpoint_-_Generic.yml:285]: changed on both (unisolate and isolate generic PBs )

File: Packs/MicrosoftDefenderAdvancedThreatProtection/pack_metadata.json
Comment (ssokolovich) [reply Packs/MicrosoftDefenderAdvancedThreatProtection/pack_metadata.json:17]: BTW I saw that it was added within this PR - https://github.com/demisto/content/pull/16553
Any thoughts on why it was added? @David-BMS

**Comment by julieschwartz18** [comment]
@ssokolovich - waiting for your feedback on the descriptions for [playbook-Microsoft_Defender_For_Endpoint_-_Isolate_Endpoint.yml](https://github.com/demisto/content/pull/16144/commits/f4a6b20058f4d843d28d7c2dd4da3e4520619ca0) and then will update the README file to align with the yaml.

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_7_0.md
Comment (ssokolovich) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_7_0.md:3]: @julieschwartz18 - the 'block' at the end should be changed to 'unblock'
Not sure also about the 'auto' term at the beginning. May I suggest 'This playbook will unisolate  **Microsoft Defender For endpoint** devices '  .....

**Pull request merged**
